fe-nc-news/src/ArticlePage.jsx

```diff
@@ -3,22 +3,30 @@ import {useState, useEffect} from 'react'
 import { useParams } from 'react-router-dom';
 import getArticle from './utils/GetArticle'; 
 import './articlePage.css'
+import getComments from './utils/GetComments'
 
 const ArticlePage = () => {
   const { article_id } = useParams(); 
   const [article, setArticle] = useState("");
   const [isLoading, setIsLoading] = useState(false);
+  const [comments, setComments] = useState([])
 
   useEffect(() => {
     setIsLoading(true);
     
     getArticle(article_id).then((articleData) => {
       setArticle(articleData);
-      console.log(article)
       setIsLoading(false);
     });
   }, [article_id]);
 
+useEffect(()=>{
+    getComments(article_id).then((commentData)=>{
```

Comment:
This should probably be in its own component as we could reasonably use comments elsewhere. Try creating a comments component and use this useEffect in there :)